DependencyInjection/SonataDashboardExtension.php

```diff
@@ -57,6 +58,9 @@ public function registerParameters(ContainerBuilder $container, array $config)
         $container->setParameter('sonata.dashboard.admin.block.entity', $config['class']['block']);
         $container->setParameter('sonata.dashboard.admin.dashboard.entity', $config['class']['dashboard']);
 
+        $container->setParameter('sonata.dashboard.admin.dashboard.templates.compose', $config['templates']['compose']);
+        $container->setParameter('sonata.dashboard.admin.dashboard.templates.compose_container_show', $config['templates']['compose_container_show']);
```

Comment:
I don't mean linebreak all of them. Linebreak any piece of text as soon as it is longer than 120 chars. This goes for every piece of text, except class declarations and method signatures, to be consistent with Symfony.